src/app/overview/overview.component.html

```diff
@@ -1,34 +1,111 @@
 <div class="row d-flex justify-content-center">
-    <h2 class="m-5">Architecture</h2><br>
+    <h2 class="m-5">Why Goblint?</h2><br>
 </div>
 <div class="row d-flex justify-content-center mb-5">
-    <div class="col"><p> 
-        Goblint is implemented in OCaml and uses an updated fork of CIL as its parser frontend for the C language.
-        The architecture of Goblint is designed to be modular. 
-        Analyses, which are defined by their abstract domains and transfer functions, can be activated via runtime configuration options. 
-        A flexible query system allows for communication between analyses. 
-        Together, the combined analyses and the control-flow graphs of the functions in the program provide the side-effecting constraint system, 
-        which is solved by some local generic solver. 
-        Post-processing the solution yields results for the analysis.
+    <div class="col"><p>
+        Developing software that is free of bugs is a complex and challenging task, and even more so in a multi-threaded setting.
+        Goblint supports developers in this endeavor, by providing insights into the code and <b>warning about potential programming errors</b>.
+        This detection of possible programming errors goes way beyond what linters typically can provide: Instead, Goblint carefully considers the semantics
+        of the entire program to flag <b>all</b> the bugs that could occur.
+    </p><p>
+        Goblint builds on <b>over a decade's worth of development and research</b> in the field of Static Analysis by Abstract Interpretation, leveraging the expertise of the contributing
+        research groups at the University of Tartu and at Technical University of Munich on <b>Abstract Interpretation, the semantics of the C programming language, Compiler Engineering,
+        and Efficient Fixpoint Computation</b> to create an analyzer targeting real-world applications.
+    </p>
+    <p> It also serves as the proving ground of our research ideas, so it is constantly being enhanced with new and exciting ideas.</p>
+    <p>
+        After over a decade of engineering, Goblint is a <b>robust framework</b> that can handle real-world applications and not just toy examples.
```

Comment:
This started off very well, but I'm not sure these internal stuff is that relevant?

```suggestion
        Goblint is not only a research platform advancing the state-of-the-art in Sound Static Analysis, but after over a decade of engineering, it is <b>robust framework</b> that can analyze real-world applications.
```